ppr-ui/tests/unit/RegisteringParty.spec.ts

```diff
@@ -0,0 +1,84 @@
+// Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { getVuexStore } from '@/store'
+import CompositionApi from '@vue/composition-api'
+import { mount, createLocalVue, Wrapper } from '@vue/test-utils'
+import { mockedRegisteringParty1 } from './test-data'
+
+// Components
+import { RegisteringParty } from '@/components/parties'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+// Events
+
+// Input field selectors / buttons
+
+/**
+ * Creates and mounts a component, so that it can be tested.
+ *
+ * @returns a Wrapper<Debtors> object with the given parameters.
+ */
+function createComponent (
+): Wrapper<any> {
+  const localVue = createLocalVue()
+  localVue.use(CompositionApi)
+  localVue.use(Vuetify)
+  document.body.setAttribute('data-app', 'true')
+  return mount(RegisteringParty, {
+    localVue,
+    propsData: {},
+    store,
+    vuetify
+  })
+}
+
+describe('RegisteringParty tests', () => {
+  let wrapper: Wrapper<any>
+
+  beforeEach(async () => {
+    wrapper = createComponent()
+  })
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders with default values', async () => {
+    expect(wrapper.findComponent(RegisteringParty).exists()).toBe(true)
+  })
+})
+
+describe('RegisteringParty store tests', () => {
+  let wrapper: Wrapper<any>
+
+  beforeEach(async () => {
+    await store.dispatch('setAddSecuredPartiesAndDebtors', {
+      registeringParty: mockedRegisteringParty1
+    })
+    wrapper = createComponent()
+  })
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders registering party table and headers', async () => {
+    expect(wrapper.find('.registering-table').exists()).toBeTruthy()
+  })
+
+  it('displays the correct rows when data is present', () => {
+    const rowCount = wrapper.vm.$el.querySelectorAll('.v-data-table .registering-row').length
+
+    expect(rowCount).toEqual(1)
+  })
+
+  it('displays the correct data in the table rows', () => {
+    const item1 = wrapper.vm.$el.querySelectorAll('.v-data-table .registering-row')[0]
+
+    expect(item1.querySelectorAll('td')[0].textContent).toContain('ABC REGISTERING')
+    expect(item1.querySelectorAll('td')[1].textContent).toContain('1234 Fort St.')
+  })
+})
```

Comment:
Perhaps add a few lines to the PartySummary test?

Reply:
Will do.